pandas-stubs/core/frame.pyi

```diff
@@ -1330,17 +1329,15 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
     def stack(
         self,
         level: IndexLabel = ...,
-        dropna: _bool = ...,
-        sort: _bool = ...,
-        future_stack: Literal[False] = ...,
+        future_stack: Literal[True] = ...,
```

Comment:
```suggestion
        future_stack: Literal[True],
```
If they specify `True`, then the other args aren't needed.  The `...` says default is `True`